closure/goog/dom/dom.js

```diff
@@ -88,7 +88,7 @@ goog.dom.getDomHelper = function(opt_element) {
 
 /**
  * Cached default DOM helper.
- * @type {goog.dom.DomHelper}
+ * @type {!goog.dom.DomHelper}
```

Comment:
It's assigned upon calling goog.dom.getDomHelper():

https://github.com/Dominator008/closure-library/blob/domHelper/closure/goog/dom/dom.js#L79-L85

Making it !goog.dom.DomHelper | undefined still triggers an NTI warning,
```
../closure-library/closure/goog/dom/dom.js:82: WARNING - Returned type does not match declared return type.
declared : goog.dom.DomHelper
found    : goog.dom.DomHelper|undefined
```
although one can argue that NTI should have inferred `goog.dom.defaultDomHelper_` is not `undefined` in this case.